app/controllers/auth.js

```diff
@@ -32,7 +35,7 @@ utils.promisify( googlePlus.people );
  */
 var init = function init() {
 
-  passport.use( new passportHttpBearer.Strategy( {}, findPersonByAccessToken ) );
+  passport.use( new passportHttpBearer.Strategy( {}, authenticate ) );
 
   router.post( '/auth/sign-in',
     middleware.validateBody,
```

Comment:
I guess, but the scopes are done through a bearer token, and this doesn't require one. Not sure…